tests/suite-unit/PluginFlyvemdmTaskstatus.php

```diff
@@ -32,34 +32,30 @@
 namespace tests\units;
 
 use Glpi\Test\CommonTestCase;
-use PluginFlyvemdmPolicy;
-use PluginFlyvemdmPolicyFactory;
 
+class PluginFlyvemdmTaskstatus extends CommonTestCase {
 
-class PluginFlyvemdmPolicyInteger  extends CommonTestCase {
-   public function beforeTestMethod($method) {
-      $this->resetState();
-      parent::beforeTestMethod($method);
-      $this->setupGLPIFramework();
-      $this->login('glpi', 'glpi');
+   /**
+    * @return object
+    */
+   private function createInstance() {
+      $this->newTestedInstance();
+      return $this->testedInstance;
    }
```

Comment:
Why no directly call $this->newTestedInstance ?

Reply:
because it's not not for typing $this->testedInstance in every test.